notebooks/Notebook1.md

```diff
@@ -1,34 +1,61 @@
+
+Images of plan for data structure:
+
+![IMG_8536](https://user-images.githubusercontent.com/97985236/229011899-63f02c5f-6cd8-45f6-b733-30941eff8a9f.jpeg)
+
+![IMG_8537](https://user-images.githubusercontent.com/97985236/229011930-5868f4ad-4787-41de-8b9b-c768a75b8c80.jpeg)
+
 
 ## Questions
 
 **What is the most pressing issue for your project? What design decision do
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
+My most pressing issue at this time is making sure that I have my repo set up correctly for java. As of right now I used a tutorial to set up a repo for all of my code and it runs. I am just not sure it is correct. The next problem will be working to build the bridge from my parser to the data structre aspect of my project. 
```

Comment:
Let me know if you want to talk about the repo at all, or if there is something specific I can look at.